.gitignore

```diff
@@ -22,5 +22,3 @@ main
 *.iml
 .vscode/
 !.vscode/launch.json
-
-pdc
```

Comment:
why were these lines removed? these prevent a binary built with `go build` accidentally being committed

Reply:
oop - I saw your mention in the description.

I think it would be more accurate as `/pdc`